ubuntu/Dockerfile

```diff
@@ -14,6 +14,19 @@ COPY dist/buildagent /opt/buildagent
 RUN useradd -m buildagent && \
     chmod +x /run-agent.sh /run-services.sh && sync
 
+ENV GOSU_VERSION 1.10
+RUN set -x \
+    && wget -O /usr/local/bin/gosu "https://github.com/tianon/gosu/releases/download/$GOSU_VERSION/gosu-$(dpkg --print-architecture)" \
+    && wget -O /usr/local/bin/gosu.asc "https://github.com/tianon/gosu/releases/download/$GOSU_VERSION/gosu-$(dpkg --print-architecture).asc" \
+    && export GNUPGHOME="$(mktemp -d)" \
+    && gpg --keyserver ha.pool.sks-keyservers.net --recv-keys B42F6819007F00F88E364FD4036A9C25BF357DD4 \
```

Comment:
Does this step still work for you?  This fails with keyserver communications error (below).  Not sure if this is a regional issue (if so are there mirrors for this keyserver) or if you are able to validate this is still successful ?  

```
gpg: keyserver communications error: keyserver unreachable
gpg: keyserver communications error: public key not found
gpg: keyserver receive failed: public key not found
```

Reply:
Yes, it does for me, let me check if ubuntu apt key servers have this key.

Reply:
Cold you please check if following works for you:
1. `gpg --keyserver keyserver.ubuntu.com --recv-keys B42F6819007F00F88E364FD4036A9C25BF357DD4`
2. `apt-key adv --keyserver keyserver.ubuntu.com --recv-keys B42F6819007F00F88E364FD4036A9C25BF357DD4`

It is weird, that it can't connect; keyserver.ubuntu.com is part of sks pool as well.
Are you able to resolve domain `dig +short ha.pool.sks-keyservers.net`?